src/order-book/api.spec.ts

```diff
@@ -1,31 +1,27 @@
-import { ethers } from 'ethers'
 import fetchMock, { enableFetchMocks } from 'jest-fetch-mock'
-import { CowSdk } from '../CowSdk'
-import { BUY_ETH_ADDRESS, OrderKind, SigningScheme } from '@cowprotocol/contracts'
-import { FeeQuoteParams, PriceQuoteLegacyParams } from '../api/cow/types'
-import QuoteError from '../api/cow/errors/QuoteError'
-import { CowError } from '../utils/common'
-import { OrderCancellation, OrderCreation, UnsignedOrder } from '../utils/sign'
-import { ZERO_ADDRESS } from '../constants'
-import { SupportedChainId } from '../constants/chains'
+import { BUY_ETH_ADDRESS } from '@cowprotocol/contracts'
+import { CowError } from '../common/cow-error'
+import { SupportedChainId } from '../common/chains'
+import { OrderBookApi } from './api'
+import { BuyTokenDestination, EcdsaSigningScheme, OrderKind, SellTokenSource, SigningScheme } from './generated'
+import { OrderCancellation as OrderCancellationGp } from '@cowprotocol/contracts/lib/esm/order'
```

Comment:
This import looks odd.
Isn't it exposed in a more central way?

Reply:
Thanks!
Fixed in https://github.com/cowprotocol/cow-sdk/pull/102